coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
Make it a priority to figure out why the coverage files are still being included in git pushes even though the coverage file is in gitignore.